templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -1,10 +1,9 @@
 {% import "./license.njk" as license %}{{license.license()}}
 import * as gax from 'google-gax';
 import * as path from 'path';
-import * as protobuf from 'protobufjs';
 
 import * as packageJson from '../../package.json';
-import * as protos from '../../pbjs-genfiles/protos';
+import * as protosType from '../../protos/protos';
```

Comment:
nit: let's make it `protoTypes` (a little bit more logical since it there are multiple types there)